ui/app/src/views/ViewDashboard.tsx

```diff
@@ -31,6 +31,23 @@ import { useDatasourceApi } from '../model/datasource-api';
 // https://apache.github.io/echarts-handbook/en/concepts/style/#theme
 const ECHARTS_THEME_OVERRIDES = {};
 
+interface DashboardBreadcrumbsProps {
+  dashboardName: string;
+  dashboardProject: string;
+}
+
+function DashboardBreadcrumbs(props: DashboardBreadcrumbsProps) {
```

Comment:
maybe move this to app/src/components?

Reply:
Yeah agreed, moving this out of ViewDashboard makes sense, I do like how you used the MUI Breadcrumbs component!